include/glow/IR/IR.h

```diff
@@ -120,7 +126,10 @@ class Instruction : public Value {
   Instruction(const Instruction &I) = delete;
   Instruction &operator=(const Instruction &I) = delete;
 
-protected:
+  /// Destroy an instruction.
```

Comment:
Does this deallocate the instruction? Does it remove it from the list or do you need to remove it before?